pkgs/development/python-modules/pook/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, aiohttp
+, buildPythonPackage
+, fetchFromGitHub
+, furl
+, isPy3k
+, jsonschema
+, mock
+, nose
+, pytestCheckHook
+, requests
+, xmltodict
+}:
+
+buildPythonPackage rec {
+  pname = "pook";
+  version = "1.0.0";
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "h2non";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1s2c0g4jis1i6sc2db1v6cr9faib6bcpjlvwnc866z45vsf70lvy";
+  };
+
+  # Separate development and examples from testing requirements
+  patches = [ ./requirements.patch ];
```

Comment:
Is this really required? Usually you only give it the checkInputs that are required for the calls we want to make. No need to patch requirements like this IMO.

Reply:
No, it's not. Don't know why I added in the first place. Removed.